game-core/src/main/java/games/strategy/engine/data/BattleRecordsList.java

```diff
@@ -73,6 +73,10 @@ public String toString() {
   }
 
   // Interpretation stuff below
+
+  /**
+   * Returns the TUV damage caused by {@code attacker} from the battle results in {@code brl}.
```

Comment:
can we fix `brl` while we are here? `brl` should be spelled out `battleRecordsList` - as a comment it would not be useful on its own